draft-ietf-rtcweb-jsep.xml

```diff
@@ -405,6 +405,29 @@ setRemote(ANSWER) |                                   |
         APIs to allow the application to control various session parameters,
         which will provide the necessary information to the browser about what
         sort of SessionDescription to produce.</t>
+
+        <section title="CNAME Semantics" anchor="sec.cname">
+          
+	      <t> RTP CNAME values provide a canonical name for the RTP
```

Comment:
This still seems like it needs some editorial cleanup. Let's try to merge the rest and then whoever has some cycles and finish tweaking this.